src/components/LayoutHeader/Nav/styles.module.css

```diff
@@ -4,6 +4,10 @@
   flex-direction: row;
   align-items: center;
 
+  ul {
+    list-style-type: none;
```

Comment:
dvc.org nav in Chrome's Auto Dark Mode:

![image](https://user-images.githubusercontent.com/43496356/151808218-6f474a53-2440-43c6-a821-f341f0b6594b.png)

PR nav: 
![image](https://user-images.githubusercontent.com/43496356/151808894-467570a8-fa77-489d-a764-3e40d7137231.png)